client/src/views/SearchResult.vue

```diff
@@ -3,5 +3,17 @@
     <h1>
         This is Search Results
     </h1>
+    <News/>
     </div>
 </template>
+<script>
+import News from '@/components/Newsletter.vue'
+export default {
+    name:'Search',
+    components:{
+        News
+    }
+
```

Comment:
lint error?

Reply:
There are quite a lint errors in the app, I'll resolve them all asap. :dagger:

Reply:
FIXED